File: NetBeans IDE/Evaluacion 2/PRACTICAS 1-10/EVA2_PRACTICA_6/Python/main.py
print("Coordenada en X = ")
iX = int(input())
print("Coordenada en Y = ")
iY = int(input())
if iX > 0 and iY > 0:
    print("Coordenadas del Cuadrante 1")
else:
    if iX < 0 and iY > 0:
        print("Coordenadas del Cuadrante 2")
    else:
        if iX < 0 and iY < 0:
            print("Coordenadas del Cuadrante 3")
        else:
            if iX > 0 and iY < 0:
                print("Coordenadas del Cuadrante 4")

File: NetBeans IDE/Evaluacion 2/PRACTICAS 1-10/EVA2_PRACTICA_6/C++/main.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cstdlib>
#include <cmath>

using namespace std;

// Headers
string toString (double);
int toInt (string);
double toDouble (string);

int main() {
    int iX, iY;
    
    cout << "Coordenada en X = " << endl;
    cin >> iX;
    cout << "Coordenada en Y = " << endl;
    cin >> iY;
    if (iX > 0 && iY > 0) {
        cout << "Coordenadas del Cuadrante 1" << endl;
    } else {
        if (iX < 0 && iY > 0) {
            cout << "Coordenadas del Cuadrante 2" << endl;
        } else {
            if (iX < 0 && iY < 0) {
                cout << "Coordenadas del Cuadrante 3" << endl;
            } else {
                if (iX > 0 && iY < 0) {
                    cout << "Coordenadas del Cuadrante 4" << endl;
                }
            }
        }
    }
    return 0;
}

// The following implements type conversion functions.
string toString (double value) { //int also
    stringstream temp;
    temp << value;
    return temp.str();
}

int toInt (string text) {
    return atoi(text.c_str());
}

double toDouble (string text) {
    return atof(text.c_str());
}
